### PR TITLE
Use eventual assertion in ServerIT

### PR DESCRIPTION
**Pull request by kokosing**
Use eventual assertion in ServerIT

### CODE REVIEW

File: presto-server-rpm/src/test/java/io/prestosql/server/rpm/ServerIT.java [hunk 1]
```diff
@@ -84,7 +85,12 @@ private static void testServer(String baseImage, String rpmHostPath, String expe
                     .waitingFor(forLogMessage(".*SERVER STARTED.*", 1).withStartupTimeout(Duration.ofMinutes(5)))
                     .start();
 
-            assertServer(container.getContainerIpAddress(), container.getMappedPort(8080), expectedJavaVersion);
+            assertEventually(
```
Comment (findepi) [inline presto-server-rpm/src/test/java/io/prestosql/server/rpm/ServerIT.java]: let's split the assertion --
port 8080 should be avail. immediately, not eventually

(i think `SHOW CATALOGS` should respond immediately too)

**Comment by electrum** [comment]
This failed for me the first time I tried in IntelliJ. I'm wondering if something changed recently?

**Comment by kokosing** [comment]
Do you suggest I should have change coordinator server to explicitly wait until it registers as worker before printing a message that server is started? The window is very short and I am not sure if it is a real problem. In real installations workers joins independently and it may take much longer for any worker to join and so no query that accesses data can be executed till then. so it seems natural that we should wait for workers after server is started. 

Maybe instead wrapping all the code I should just wait for worker in eventual assertion. WDYT?

**Comment by electrum** [comment]
You reminded me of the `query-manager.required-workers` config, which should solve this. But I'd like to know why the coordinator is not already registered, since the service selector should merge announcements from the local announcer.

**Comment by kokosing** [comment]
I think it is because the query against jmx. Other queries should be passing just fine. And I added that query recently.

**Comment by electrum** [comment]
There must be something else wrong because `query-manager.required-workers` already defaults to `1` yet it is still failing.

File: presto-server-rpm/src/test/java/io/prestosql/server/rpm/ServerIT.java (hunk 1 above)
Comment (kokosing) [reply presto-server-rpm/src/test/java/io/prestosql/server/rpm/ServerIT.java]: > let's split the assertion --

It used to be splitted. :)

**Comment by kokosing** [comment]
I updated the pull request.

**Comment by kokosing** [comment]
The potential problem that was found here is not much related to rpm tests. Also it is tracked under separate issue and there is as TODO added into code. Therefore I am going to merge this pull request soon to avoid false negative tests results

**Pull request merged**